### DISCUSSION TITLE
I'm getting this error: pg_restore: error: input file appears to be a text format dump. Please use psql.

### DISCUSSION THREAD

**Discussion by alcarazolabs** [spatie/laravel-backup#1622] [General] [answered]
Hi dear,
After create one backup by using laravel backup panel with uses this package "laravel-backup" and importing it by using pgadmin because my database is a postgresql it throwed one error:
pg_restore: error: input file appears to be a text format dump. Please use psql.
My DB_CONNECTION is =pgsql
I have installed laravel backup  spatie v6.
By the moment the backups are being done successfully but when I try to restore a database by using this backup with pgadmin v4 it is throwing the error of above.

I don't know what to do, in local mode I'm working with mysql database but in production I'm using postgresql.
This issue is with this package or with Laravel-backup from spatie.

I will appreciate any idea to fix this problem.
thanks.
![image](https://user-images.githubusercontent.com/7144988/206581983-d4d1e9e3-71c0-427d-b96e-b88a3c359239.png)

**Comment by parallels999** [comment, marked as answer]
Use psql

**Reply by alcarazolabs** [reply]
> Use psql

I used
```
 'dump' => [
            'add_extra_option' => '--format=c', // and any other pg_dump flags
       ]
```
And fixed it.
